resemble.js

```diff
@@ -478,6 +478,16 @@ var _this = {};
 					if(hasMethod) { param(); }
 					return self;
 				},
+				setTolerance: function(value){
+
+					tolerance.red = value;
+					tolerance.green = value;
+					tolerance.blue = value;
+					tolerance.alpha = value;
+
```

Comment:
The values are initialized to the same values as ```ignoreNothing```.  There isn't a clear requirement for these functions to be exclusive operations, and I'm intending for setTolerance to potentially be combined with other functions.  For example, in our code, currently a tolerance level of 37 is required.  If we decide that ```ignoreAntialiasing``` is also required in some scenarios, then we will be able to both ignore antialiasing and increase the tolerance.